app/src/main/java/io/github/sustainow/presentation/ui/components/MultiFieldQuestionCard.kt

```diff
@@ -0,0 +1,490 @@
+                            )
+                            if (isRealConsumption) {
+                                Box(
+                                    modifier = Modifier
+                                        .height(24.dp)
+                                        .aspectRatio(1f)
+                                        .clickable {
+                                            isEditSubItem = !isEditSubItem
+                                        },
+                                    contentAlignment = Alignment.Center
+                                ) {
+                                    Icon(
+                                        Icons.Outlined.Edit,
+                                        tint = MaterialTheme.colorScheme.onSurfaceVariant,
+                                        contentDescription = null,
+                                        modifier = Modifier
+                                            .size(12.dp)
+                                            .aspectRatio(1f)
+                                    )
+                                }
+                            }
+                        }
+
+                        Spacer(modifier = Modifier.weight(1f))
+
+                        Row (
+                            horizontalArrangement = Arrangement.Center,
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            BasicTextField(
+                                modifier = Modifier
+                                    .height(28.dp)
+                                    .width(58.dp),
+                                textStyle = TextStyle(
+                                    fontSize = 8.sp,
+                                    textAlign = TextAlign.Center,
+                                ),
+                                decorationBox = { innerTextField ->
+                                    Box(
+                                        modifier = Modifier
+                                            .border(
+                                                1.dp,
+                                                MaterialTheme.colorScheme.outline,
+                                                MaterialTheme.shapes.small
+                                            ),
+                                        contentAlignment = Alignment.Center
+                                    ) {
+                                        innerTextField()
+                                    }
+                                },
+                                value = item.timePeriod.toString(),
+                                onValueChange = { newTime ->
+                                    val updateTime =
+                                        if (newTime.isBlank()) Duration.ZERO else Duration.parse(
+                                            newTime
+                                        )
```

Comment:
Não existe um manipulamento de erros, aplicação crasha se a duração não for válida. Mostrar supportingText de apoio quando a duração não for válida